packages/nestjs-auth-google/src/exceptions/auth-google-missing-id.exception.ts

```diff
@@ -0,0 +1,8 @@
+import { RuntimeException } from '@concepta/nestjs-exception';
+
+export class AuthGoogleMissingIdException extends RuntimeException {
+  constructor(message = 'Google did not return an id for the user.') {
+    super(message);
```

Comment:
This will default to 500... should add a friendly error message. I can't remember if it does this by default.